src/Components/OrganizationData/OrganizationDataLayout/OrganizationDataLayout.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import  { Box, Container, Button} from '@material-ui/core';
+
+const OrganizationDataLayout = ({organizationData}) => {
+
+    const { push } = useHistory()
+    const hasData= !!organizationData
+    return ( 
+
+        <Container >
+            <img alt="organizationPhoto" src={hasData ? organizationData.img :"https://via.placeholder.com/350"} />
+            <div>Nombre de la Organizacion</div>
+            <Box sx={{display: 'flex'}}> 
+                {hasData 
```

Comment:
this should be a function, maybe it can be called "showOrganizationDescription()"

Reply:
Solved! Ty